vitest.config.ts

```
@@ -0,0 +1,19 @@
+import { defineConfig } from "vitest/config";
```

ESLint

[error] Replace `"vitest/config"` with `'vitest/config'`

(prettierPlugin/prettier)